tests/integration/suite/test_auth_configs.py

```diff
@@ -110,3 +110,18 @@ def key_secret_creation(k8sclient):
         if s.metadata.name == "pingconfig-spkey":
             return True
     return False
+
+
+def test_auth_label(admin_mc, user_factory):
+    user = user_factory()
+    k8s_client = CustomObjectsApi(admin_mc.k8s_client)
+    tokens = k8s_client.list_cluster_custom_object(
```

Comment:
There should be a wait_for in here to ensure we actually get the users token before attempting to access it.

Reply:
Fixed now. Also re-vendored types, which included another change but it looks legit.